core/domain/acl_decorators_test.py

```diff
@@ -351,18 +351,18 @@ def test_normal_user_can_access_editor_dashboard(self):
         self.assertEqual(response.status_int, 200)
 
 
-class ViewExplorationFeedbackTest(test_utils.GenericTestBase):
-    """Tests for can_view_exploration_feedback decorator."""
+class CommentOnFeedbackTest(test_utils.GenericTestBase):
```

Comment:
Where are all the tests for the new functions that's been added to acl_decorators? This is important fundamental code; it should be very well-covered by tests to root out all the edge cases.